eblocker-icapserver/src/main/java/org/eblocker/server/common/data/Device.java

```diff
@@ -82,8 +82,13 @@ public class Device extends ModelObject {
     private boolean controlBarAutoMode = true;
     private boolean mobileState = true;
     private boolean mobilePrivateNetworkAccess;
+    private String lastSeen = "-";
```

Comment:
Shouldn't this be better an absolute timestamp (i.e. java.time.Instant) instead of a string? I think it would be easier to persist this data in the database.

Currently a number days is stored in `lastSeen`. This might get out of sync if the eBlocker is switched off for some days.

Reply:
implemented in 226816c462a1663efe2c0ef925f029f9e5369be7
but saved in a string because angular doesn't support calling functions